meson.build

```diff
@@ -580,6 +580,14 @@ vcs_tagger = [meson.source_root() + '/tools/meson-vcs-tag.sh',
               get_option('version-tag'),
               meson.project_version()]
 
+version_h = vcs_tag(
+        input : 'src/version/version.h.in',
+        output : 'version.h',
+        command: vcs_tagger,
+        fallback: meson.project_version())
```

Comment:
This fallback line shouldn't be necessary, but looks good otherwise.

Reply:
Yeah, please remove this line, since vcs-tagger already implements the fallback and there's no reason to duplicate it here.

Reply:
@keszybz removed